packages/next/build/analysis/get-page-static-info.ts

```diff
@@ -313,17 +313,13 @@ export async function getPageStaticInfo(params: {
       !isEdgeRuntime(resolvedRuntime)
     ) {
       const options = Object.values(SERVER_RUNTIME).join(', ')
-      if (typeof resolvedRuntime !== 'string') {
-        Log.error(
-          `The \`runtime\` config must be a string. Please leave it empty or choose one of: ${options}`
-        )
-      } else {
-        Log.error(
-          `Provided runtime "${resolvedRuntime}" is not supported. Please leave it empty or choose one of: ${options}`
-        )
-      }
+      const message =
+        typeof resolvedRuntime !== 'string'
+          ? `The \`runtime\` config must be a string. Please leave it empty or choose one of: ${options}`
+          : `Provided runtime "${resolvedRuntime}" is not supported. Please leave it empty or choose one of: ${options}`
+      Log.error(message)
```

Comment:
We probably want to remove the `Log.error` if we're gonna throw since it will be duplicated in the output otherwise.

Reply:
I'll keep it for dev mode (where we don't throw nor exit), and remove it for non-dev. Same with the one bellow.